src/troubleshooting.ts

```diff
@@ -2,21 +2,47 @@
 
 import {
   CancellationToken,
+  ColorThemeKind,
+  commands,
   ExtensionContext,
   Uri,
   WebviewView,
   WebviewViewProvider,
   WebviewViewResolveContext,
-  commands,
+  window,
 } from "vscode";
 import { randomBytes } from "crypto";
+import { marked } from "marked";
+import sanitizeHtml from "sanitize-html";
+
+// NOTE: marked code block formatting relies on the `marked.use` block in `chat.ts`
 
-type TroubleshootingResult = any;
+type TroubleshootingResult = {
+  type: "error" | "assistance" | "feedback" | "user" | string;
```

Comment:
having string here seems a little odd

Reply:
Removed in the next PR, I was kind of thinking "this is a free entry type but these are the expected ones"